src/Directory.Build.props

```diff
@@ -5,6 +5,9 @@
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)stylecop.ruleset</CodeAnalysisRuleSet>
     <NoWarn>NU1701</NoWarn>
     <TreatWarningsAsErrors Condition="'$(Configuration)'=='Release'">true</TreatWarningsAsErrors>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
According to https://github.com/dotnet/sourcelink, the recommending way is to create a .snuget package. Have we considered that?

Reply:
https://github.com/dotnet/docfx/commit/8da80e2b477fb003475a57d109485e57c53a8e4e